File: next/security/backend-ms1/src/main/java/ma/zs/stocky/ws/dto/security/RoleUserDto.java
package  ma.zs.stocky.ws.dto.security;

import ma.zs.stocky.zynerator.audit.Log;
import ma.zs.stocky.zynerator.dto.AuditBaseDto;
import com.fasterxml.jackson.annotation.JsonInclude;





@JsonInclude(JsonInclude.Include.NON_NULL)
public class RoleUserDto  extends AuditBaseDto {


    private UserDto user ;
    private RoleDto role ;



    public RoleUserDto(){
        super();
    }




    public UserDto getUser(){
        return this.user;
    }

    public void setUser(UserDto user){
        this.user = user;
    }
    public RoleDto getRole(){
        return this.role;
    }

    public void setRole(RoleDto role){
        this.role = role;
    }






}

File: laravel/security/backend-ms1/app/Dto/security/RoleUserDto.php
<?php

namespace App\Dto\security;


class RoleUserDto
{

    private $user,
    private $role,




    public function getId() {
        return $this->id;
    }
    public function setId($id){
        $this->id = $id;
    }

    public function getUser() {
        return $this->user;
    }
    public function setUser($user){
        $this->user = $user;
    }

    public function getRole() {
        return $this->role;
    }
    public function setRole($role){
        $this->role = $role;
    }

}


